core/shared/src/main/scala/org/scalacheck/Test.scala

```diff
@@ -369,7 +369,7 @@ object Test {
       val help = "Maximum number of RNG spins to perform between checks"
     }
 
-    val opts = Set[Opt[_]](
+    val opts: collection.Set[Opt[_]] = Set[Opt[_]](
```

Comment:
Just wondering – can the type be `collection.immutabe.Set` rather than `collection.Set`? Or does it break compatibility?